packages/cloud-cognitive/src/components/Datagrid/Datagrid.test.js

```diff
@@ -133,4 +733,471 @@ describe(componentName, () => {
       }
     );
   });
+
+  //Empty State
+  it('renders an empty table', () => {
```

Comment:
The "Empty State" test case has covered the Datagrid code that handles the empty state, which is great. However, the only verification performed is the check of the class name in the `tbody` element. It would be really nice if you can add more verification such as checking there is in fact no rows in the table, or anything like that.